web/src/click_handlers.js

```diff
@@ -384,16 +384,15 @@ export function initialize() {
         message_edit.toggle_resolve_topic(message_id, topic_name);
     });
 
-    // TOPIC MUTING
-    $("body").on("click", ".message_header .on_hover_topic_mute", (e) => {
-        e.stopPropagation();
-        muted_topics_ui.mute_or_unmute_topic($(e.target), true);
-    });
-
-    $("body").on("click", ".message_header .on_hover_topic_unmute", (e) => {
-        e.stopPropagation();
-        muted_topics_ui.mute_or_unmute_topic($(e.target), false);
-    });
+    // TOPIC MUTING/UNMUTING
+    $("body").on(
+        "click",
+        ".message_header .on_hover_topic_toggled,.message_header .on_hover_topic_toggle",
+        (e) => {
+            e.stopPropagation();
+            muted_topics_ui.mute_or_unmute_topic($(e.target));
+        },
```

Comment:
`on_hover_topic_toggled` and `on_hover_topic_toggle` are unclear class names. Let's use `toggle_topic_muted` for both of them and apply CSS based on the icon visible.